plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftClientExtension.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * ***************************************************************************** Copyright (c)
```

Comment:
headers seem broken

Reply:
Fixed header's in OpenShiftClientExtension and OpenShiftVersion.